Python/sysmodule.c

```diff
@@ -125,11 +125,10 @@ sys_set_object(PyInterpreterState *interp, PyObject *key, PyObject *v)
     }
     PyObject *sd = interp->sysdict;
     if (v == NULL) {
-        v = _PyDict_Pop(sd, key, Py_None);
-        if (v == NULL) {
+        if (PyDict_Pop(sd, key, &v) < 0) {
```

Comment:
NULL can be passed here.

Reply:
Oh, I used NULL but the change got lost somehow: fixed.